evaluate.py

```diff
@@ -38,32 +32,9 @@ def evaluate(config, model, dataset_loader, eval_metric, split='val', dump=True)
     return metrics
 
 
-class FewShotMetrics():
-    def __init__(self, config, dataset=None, few_shot_limit=5, target_label='label'):
-        # read train / test labels
-        # dev is not considered for now
-        # test_labels = np.hstack([instance[target_label]
-                                 # for instance in dataset['test']])
-        # train_labels = np.hstack([instance[target_label]
-                                  # for instance in dataset['train']])
-
+class MultiLabelMetrics():
+    def __init__(self, config, dataset):
```

Comment:
The metric doesn't need a dataset now.
```suggestion
    def __init__(self, config):
```